pgamit/pyETM.py

```diff
@@ -282,9 +298,13 @@ def to_postgres(dictionary):
     if isinstance(dictionary, dict):
         for key, val in list(dictionary.items()):
             if isinstance(val, np.ndarray):
-                dictionary[key] = str(val.flatten().tolist()).replace('[', '{').replace(']', '}')
+                dictionary[key] = \
+                    str(val.flatten().tolist()).replace(
+                        '[', '{').replace(']', '}')
     else:
-        dictionary = str(dictionary.flatten().tolist()).replace('[', '{').replace(']', '}')
+        dictionary = \
+            str(dictionary.flatten().tolist()).replace(
+                '[', '{').replace(']', '}')
```

Comment:
parens continuation is generally preferred over backslashes, but these statements are hard to truncate in a readable way... my suggestion would be to split on an empty function parens.

So either here:

```python
def to_postgres(dictionary):
    if isinstance(dictionary, dict):
        for key, val in list(dictionary.items()):
            if isinstance(val, np.ndarray):
                dictionary[key] = str(val.flatten().tolist(
                    )).replace('[', '{').replace(']', '}')
    else:
        dictionary = str(dictionary.flatten().tolist(
            )).replace('[', '{').replace(']', '}')

    return dictionary
```

or here:
```python
def to_postgres(dictionary):
    if isinstance(dictionary, dict):
        for key, val in list(dictionary.items()):
            if isinstance(val, np.ndarray):
                dictionary[key] = str(val.flatten().tolist()).replace(
                        '[', '{').replace(']', '}')
    else:
        dictionary = str(dictionary.flatten().tolist()).replace(
                '[', '{').replace(']', '}')

    return dictionary
```